src/components/header.js

```diff
@@ -21,12 +21,15 @@ function Header() {
     if (path !== '/')
       return (
         <OrangeFCCLogo
-          className="logo"
-          alt={`${site.siteMetadata.title} Logo`}
+          className="logo logoOrange"
+          alt={`${site.siteMetadata.title} Orange Logo`}
```

Comment:
What do you think about changing this name in the same way I commented on line 30?